test/test_meta.py

```diff
@@ -154,25 +154,51 @@ def setUpClass(cls):
         cls.boundaries = cls.by_name['boundaries']
 
     def test_osm(self):
+        from shapely.geometry import Point
+        shape = Point(0, 0).buffer(1.0)
         props = {
             'boundary': 'administrative',
             'boundary:type': 'aboriginal_lands',
             'admin_level': '2',
         }
         meta = make_test_metadata()
-        out_props = self.boundaries.fn(None, props, None, meta)
+        out_props = self.boundaries.fn(shape, props, None, meta)
         self.assertEquals('aboriginal_lands', out_props.get('kind'))
         self.assertEquals('2', out_props.get('kind_detail'))
 
     def test_ne(self):
+        from shapely.geometry import Point
+        shape = Point(0, 0).buffer(1.0)
         props = {
             'featurecla': 'Admin-1 region boundary',
         }
         meta = make_test_metadata()
-        out_props = self.boundaries.fn(None, props, None, meta)
+        out_props = self.boundaries.fn(shape, props, None, meta)
         self.assertEquals('macroregion', out_props.get('kind'))
         self.assertEquals('3', out_props.get('kind_detail'))
 
+    def test_osm_linestring(self):
+        from shapely.geometry import LineString
+        shape = LineString([(0, 0), (1, 1)])
+        props = {
+            'boundary': 'administrative',
+            'boundary:type': 'aboriginal_lands',
+            'admin_level': '2',
+        }
+        meta = make_test_metadata()
+        out_props = self.boundaries.fn(shape, props, None, meta)
+
+        # without the hack, shouldn't match (i.e: as if it were from
```

Comment:
Please explain for our future selves the nature of this hack inline here in more detail (even if just `boundaries point-of-view hack`?

Reply:
Good point. Added more comprehensive comment in 095be669.